values/prometheus-operator/prometheus-operator.gotmpl

```diff
@@ -47,6 +47,8 @@ prometheusOperator:
     resources: {{- $p.resources.prometheusConfigReloader | toYaml | nindent 6 }}    
 commonLabels:
   prometheus: system
+crds:
```

Comment:
I am not sure if we can rely on it. 
The ingress-nginx-controller chart is deploying the ServiceMonitor during the very first install.
It means that crds needs to be deployed during the initial install.